app/views/components/pageflow/admin/users_tab.rb

```diff
@@ -38,7 +38,8 @@ def build(theming)
               end
             end
           end
-          add_membership_button_if_needed(current_user, account, 'account')
+          add_membership_button_if_needed(current_user, account, 'account') if
+            Pageflow.config.allow_multiaccount_users
```

Comment:
`add_membership_button_if_needed` already contains a lot of logic. I'd suggest moving this piece of logic there as well. It's also easier to test logic like the on the view component level.